src/openapi.json

```diff
@@ -348,6 +348,42 @@
           "title",
           "createdAt"
         ]
+      },
+      "Bundle": {
```

Comment:
I am a bit concerned it could be confusing to link to someone else's documentation from our documentation (since it might cause folks to be unsure of which parts of the FHIR spec we are / are not implementing, and while we are inspired by it we are not committing to follow it now or necessarily in future).

What do you think of instead maintaining that association in the developer logs / the commit message (I should include that in the commit message either way!).

Reply:
For now I've at least added the link to the comment.  Marking this as resolved BUT please un-resolve if you're feeling it's worth having more discussion about including it in the documentation directly.